packages/react-reconciler/src/ReactDebugFiberPerf.js

```diff
@@ -231,6 +232,25 @@ export function recordScheduleUpdate(): void {
   }
 }
 
+export function startRequestCallbackTimer(): void {
+  if (enableUserTimingAPI && !isWaitingForCallback) {
```

Comment:
Can we keep this nested? I want to keep all top level conditions uniform so it's obvious they all have to include the `enableUserTimingAPI` flag check.